src/reset-password/data/sagas.js

```diff
@@ -0,0 +1,51 @@
+import { call, put, takeEvery } from 'redux-saga/effects';
+
+// Actions
+import {
+  VALIDATE_TOKEN,
+  validateTokenBegin,
+  validateTokenSuccess,
+  validateTokenFailure,
+  RESET_PASSWORD,
+  resetPasswordBegin,
+  resetPasswordSuccess,
+  resetPasswordFailure,
+} from './actions';
+
+import { validateToken, resetPassword } from './service';
+
+
+// Services
+export function* handleValidateToken(action) {
+  try {
+    yield put(validateTokenBegin());
+    const { isValid } = yield call(validateToken, action.payload.token);
```

Comment:
The returned key is `is_valid` or we should change the endpoint to return `isValid` otherwise it will always be undefined.